astrix-gs/src/main/java/com/avanza/astrix/remoting/util/GsUtil.java

```diff
@@ -59,18 +59,22 @@ private static ServiceInvocationException toRuntimeException(Throwable exception
 		}
 	}
 	
-	public static <T> void subscribe(final AsyncFuture<T> asyncFuture, final Subscriber<? super T> t1) {
-		asyncFuture.setListener(new AsyncFutureListener<T>() {
-			@Override
-			public void onResult(AsyncResult<T> result) {
+	public static <T> void subscribe(
```

Comment:
Since the original method is `public` it should probably be overloaded with an extra parameter instead and the original method annotated as `@Deprecated`.

Reply:
Yes, good point. Have updated this in a separate commit now.